src/main/java/doore/study/domain/Study.java

```diff
@@ -51,4 +58,45 @@ public class Study extends BaseEntity {
     @OneToMany(mappedBy = "study")
     private List<CurriculumItem> curriculumItems;
 
+
+    public void createCurriculumItems(List<CurriculumItem> newCurriculumItems) {
+        if (curriculumItems == null) {
+            return;
+        }
+        newCurriculumItems.forEach(newCurriculumItem -> {
+            newCurriculumItem.saveStudyToCurriculum(this);
```

Comment:
```suggestion
            newCurriculumItem.saveStudy(this);
```
Nit: 메서드 명은 이렇게 해도 설명이 충분한 것 같아요~
연관관계 편의 메서드 사용 좋습니다 👍👍👍